src/main/scala/models/Opportunity.scala

```diff
@@ -26,7 +26,21 @@ case class Opportunity(
                         duplicatedFrom: Option[OpportunityId],
                         description: Seq[OpportunityDescriptionSection]
                       ) {
+
   lazy val summary: OpportunitySummary = OpportunitySummary(id, title, startDate, endDate, value, publishedAt, duplicatedFrom)
+
+  lazy val statusString: String = {
+    publishedAt.isDefined match {
+      case true if isEndDatePassed => "Closed"
+      case true if isStartDatePassed => "Open"
+      case true => "Queued"
+      case false => "Draft"
+    }
+  }
+
+  lazy val isStartDatePassed = startDate.isEqual(LocalDate.now) || startDate.isBefore(LocalDate.now)
```

Comment:
Ouch slight lack of attention to detail on the class name their [sic].  Interesting the new opportunity definition, nice touch.